common/src/main/java/com/cinemamod/mcef/CefUtil.java

```diff
@@ -87,4 +87,46 @@ static CefApp getCefApp() {
     static CefClient getCefClient() {
         return cefClientInstance;
     }
+
+    public static String mimeFromExtension(String ext) {
+//        ext = ext.toLowerCase();
+//        String ret = mimeTypeMap.get(ext);
+//        if (ret != null)
+//            return ret;
+
+        //If the mimeTypeMap couldn't be loaded, fall back to common things
```

Comment:
I think, remove these comments?

Reply:
No - this is something I want to port from the old MCEF but haven’t gotten around to and don’t yet need to